Source/Entities/Gimmicks/VerticalBar.tscn

```diff
@@ -0,0 +1,41 @@
+[gd_scene load_steps=6 format=2]
+
+[ext_resource path="res://Graphics/Gimmicks/VerticalBar.png" type="Texture" id=1]
+[ext_resource path="res://Scripts/Gimmicks/VerticalBar.gd" type="Script" id=2]
+[ext_resource path="res://Audio/SFX/Player/Grab.wav" type="AudioStream" id=3]
+
+[sub_resource type="RectangleShape2D" id=1]
+extents = Vector2( 4, 2.5 )
+
+[sub_resource type="RectangleShape2D" id=2]
+extents = Vector2( 2, 30.5 )
+
+[node name="VerticalBar" type="Node2D"]
+
+[node name="VerticalBarSprite" type="Sprite" parent="."]
+texture = ExtResource( 1 )
+
+[node name="VerticalBarTopPlatform" type="StaticBody2D" parent="."]
+collision_layer = 8193
+collision_mask = 0
+
+[node name="VerticalBarCollision" type="CollisionShape2D" parent="VerticalBarTopPlatform"]
+position = Vector2( 0, -31.5 )
+shape = SubResource( 1 )
+one_way_collision = true
+
+[node name="VerticalBarArea" type="Area2D" parent="."]
+collision_layer = 8192
+collision_mask = 8192
+script = ExtResource( 2 )
```

Comment:
Small mistake here with the collision mask with big consequences. It currently has layer_mask 14 and it really should only have collision_layer 14... leading to non-player objects trying to interact with it. This will be fixed in the next commit.